assets/app.js

```diff
@@ -984,6 +1010,52 @@ new Vue({
                 this.products[index].descriptions[language] = my_description
             }
         },
+        bulkChangeTranslationStatus: function (type) {
+            proceed = confirm("Are you sure that you want to set all " + this.resourceDict[type]['name'] + " to be fully translated?")
+
+            if (proceed && hasApi) {
+                let resource_translation_requested = this[this.resourceDict[type].translationRequested]
```

Comment:
use `let resource_translation_requested = this.resourceDict[type].translationRequested`